Backend_Repo/website/migrations/0003_hackathons_talks_workshops.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('website', '0002_auto_20170219_0158'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Hackathons',
+            fields=[
+                ('id', models.AutoField(auto_created=True, serialize=False, verbose_name='ID', primary_key=True)),
+                ('title', models.CharField(max_length=255)),
+                ('picUrl', models.CharField(max_length=255)),
+                ('date', models.DateField()),
+                ('content', models.TextField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Talks',
+            fields=[
+                ('id', models.AutoField(auto_created=True, serialize=False, verbose_name='ID', primary_key=True)),
+                ('title', models.CharField(max_length=255)),
+                ('picUrl', models.CharField(max_length=255)),
+                ('date', models.DateField()),
+                ('content', models.TextField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Workshops',
+            fields=[
+                ('id', models.AutoField(auto_created=True, serialize=False, verbose_name='ID', primary_key=True)),
+                ('title', models.CharField(max_length=255)),
+                ('picUrl', models.CharField(max_length=255)),
```

Comment:
Should we make one modal instead? Called `events` and give it a type field? (Not exactly necessary but this code is not DRY.)